testing/describes.js

```diff
@@ -220,6 +220,8 @@ export const realWin = describeEnv((spec) => [
  *   hash: (string|undefined),
  *   amp: (boolean),
  *   timeout: (number),
+ *   ifIe: (boolean),
+ *   enableIe: (boolean),
```

Comment:
While you're here cleaning up this code, I think we should allow for all `describe` config modifiers to be used on `describes` instead of just the IE-specific ones. See the full list here:

https://github.com/ampproject/amphtml/blob/a01f28121e1e090b3945af9ecca71f3f4a168c27/test/_init_tests.js#L136-L198

Maybe they could all be (optionally) passed in via a `config` object instead of individual members of `spec`? Doing so will make it possible to skip / enable an entire `describes` block for a browser. (Right now, you can only skip nested blocks inside a `describes`.)

WDYT?

Reply:
I support the general idea of this, but I think it would benefit from some time spent analyzing how these are used in context. Are entire suites typically skipped, or are they used to run different variants of the same test on different browsers within one integration suite? It's not obvious to me yet how we should map it. I suspect the ideal will be something like providing a `browsers` object in a spec, with defaults of `true` for all browsers except IE. I'd lean towards punting this to a later PR, so as to minimize the scale of changes made to test infra at any one time

Reply:
Future PR SGTM.

Reply:
Leaving this unresolved so I can come back to reference it in the follow-up

Reply:
This open thread and yesterday's follow up convo prompted me to tackle this more comprehensively in #34286.